pkgs/by-name/ta/taisei/package.nix

```
@@ -0,0 +1,104 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  # Build depends
+  docutils,
+  meson,
+  ninja,
+  pkg-config,
+  python3,
+  python312Packages,
+  ensureNewerSourcesForZipFilesHook,
+  # Runtime depends
+  glfw,
+  SDL2,
+  SDL2_mixer,
+  cglm,
+  freetype,
+  libpng,
+  libwebp,
+  libzip,
+  zlib,
+  zstd,
+  opusfile,
+  openssl,
+  gamemode,
+  shaderc,
+  spirv-cross,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "taisei";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "taisei-project";
+    repo = "taisei";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rThLz8o6IYhIBUc0b1sAQi2aF28btajcM1ScTv+qn6c=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    docutils
+    meson
+    ninja
+    pkg-config
+    python3
+    python312Packages.zstandard
```

```suggestion
    python3Packages.python
    python3Packages.zstandard
```
using `python3Packages` will always use the latest python version.
also it might be better to have a single source for python related stuff so using `python3Packages.python` is a bit better than `python3`